changelogs/fragments/1079-redis-use-regexp-to-check-if-the-value-matches-expected-form.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fixes redis parsing of config values which should not be converted to bytes
```

Comment:
```suggestion
  - redis - fixes parsing of config values which should not be converted to bytes (https://github.com/ansible-collections/community.general/pull/1079).
```

Reply:
@mjrider please don't forget about this one :)